config-connector/solutions/projects/kpt/owned-project/Kptfile

```diff
@@ -0,0 +1,36 @@
+apiVersion: kpt.dev/v1alpha1
+kind: Kptfile
+metadata:
+  name: owned-project
+packageMetadata:
+  shortDescription: Create a project under a folder and owner permission for the project
```

Comment:
Could you make the short description consistent with SYNOPSIS? The problem here is that owner is the role. So I'd say "creates a project under a folder and grants an IAM member Owner role to the project"